Documentation/Helm-Charts/ceph-cluster-chart.md

```diff
@@ -137,6 +137,12 @@ The `cephObjectStores` array in the values file will define a list of CephObject
 | `storageClass.name` | The name of the storage class | `ceph-bucket` |
 | `storageClass.parameters` | See [Object Store storage class](../Storage-Configuration/Object-Storage-RGW/ceph-object-bucket-claim.md) documentation or the helm values.yaml for suitable values | see values.yaml |
 | `storageClass.reclaimPolicy` | The default [Reclaim Policy](https://kubernetes.io/docs/concepts/storage/storage-classes/#reclaim-policy) to apply to PVCs created with this storage class. | `Delete` |
+| `ingress.enabled` | Enable an ingress for the ceph-objectstore | `false` |
```

Comment:
The docs-check CI is failing because this is not matching the above template from which it is generated.

Reply:
@jouve you need to run `make helm-docs` in the root of the repo and commit those changes as well. The Helm chart doc is auto generated.